.github/workflows/ci-python.yml

```
@@ -1,38 +1,34 @@
+---
 name: Build and test Python package
-
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
-
+    branches: [main]
 jobs:
   linting:
     name: Linting
-    runs-on: 'ubuntu-latest'
+    runs-on: ubuntu-latest
     strategy:
-        matrix:
-          python-version:
-            - 3.9
-            - "3.10"
-            - "3.11"
+      matrix:
+        python-version:
+          - '3.10'
+          - '3.11'
```

```suggestion
          - '3.11'
          - '3.12'
```
I thin we should add 3.12 to our linting and tests, because it available since half a year or sth. and some team members are already using it

---

added in d6b26dd99787cc917b0d1689616b0a2173332971